.travis.yml

```diff
@@ -23,28 +13,30 @@ addons:
 jdk:
   - oraclejdk8
 
+before_install:
+  # Install SDK license so Android Gradle plugin can install deps.
+  - mkdir "$ANDROID_HOME/licenses" || true
+  - echo "d56f5187479451eabf01fb78af6dfcb131a6481e" > "$ANDROID_HOME/licenses/android-sdk-license"
+  - echo "24333f8a63b6825ea9c5514f83c2829b004d1fee" >> "$ANDROID_HOME/licenses/android-sdk-license"
+  # Install the rest of tools (e.g., avdmanager)
+  - sdkmanager tools > /dev/null
+  # Install the emulator system image, NDK, and CMake
+  - sdkmanager "system-images;android-18;default;armeabi-v7a" ndk-bundle "cmake;3.10.2.4988404" > /dev/null
```

Comment:
redirecting stdout because https://issuetracker.google.com/issues/137389944 causes 5-10k lines of progress updates